### ISSUE TITLE
Additional tests required for MAReferencedDataWriterReader_visitors

### ISSUE THREAD

**Issue by passimens** [Assargadon/magritte-python#141]
Missing additional tests in Magritte/visitors/tests/MAReferencedDataWriterReader_visitors_test.py:

-MAReferencedDataWriterVisitorTest
--testIgnoreInvisible
--testProcessPlugableReadable
--testIgnorePlugableNonReadable
-MAReferencedDataReaderVisitorTest
--testIgnoreInvisible
--testIgnoreReadonly
--testProcessPlugableWritable
--testIgnorePlugableNonWritable

To support tests we need to enrich descriptions in Magritte/model_for_tests/ModelDescriptor_test.py with respective elements:
- invisble
- MAPlugableAccessor (can read, cannot write)
- MAPlugableAccessor (cannot read, can write)

**Comment by Assargadon** [comment]
Non-test-covered change: https://github.com/Assargadon/magritte-python/pull/140
